### PR TITLE
Features/user login updates 3529

### PR DESCRIPTION
**Pull request by kkellydesign**
This is the markup for the new login screen design. I've included an SVG version of the FF logo and the new background/foreground styles. The SUMO login form is at the top of the page, and the markup needed for the FF account screen is just below that. 

ref #3529

### CODE REVIEW

File: kitsune/users/jinja2/users/includes/login-form.html [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+
+  <br>
+  <br>
+  <br>
+  <br>
+  <h1 class="mb-0">{{ _('Log in / Sign up with Firefox Accounts') }}</h1>
+  <p class="mt-0">We are upgrading our log-in system to replace Mozilla Support accounts with Firefox Accounts. Firefox Accounts offers one log-in for convenient and secure access to Mozilla services. <a href="#TODO">See our FAQ</a> for more information.</p>
+
+  <div>
+  <a href="#TODO" class="button">Continue with Firefox Accounts</a>
```
Comment (johngian) [inline kitsune/users/jinja2/users/includes/login-form.html]: There is an implementation in a different branch:
https://github.com/mozilla/kitsune/blob/firefox-accounts-integration/kitsune/users/urls.py#L101

Maybe you can use this URL for consistency.

File: kitsune/users/jinja2/users/includes/login-form.html [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+  </div>
+  <p><a href="/users/auth">continue w/ SUMO account</a></p>
```
[the remark this answers is not in the document]
Comment (mcranston18) [reply kitsune/users/jinja2/users/includes/login-form.html]: According to the designs, `continue w/ SUMO account` should not be capitalized https://app.zeplin.io/project/5c940ea1d154600581d4377b/screen/5cbef4e9d398f81fa6770678

File: kitsune/users/jinja2/users/includes/login-form.html (hunk 1 above)
Comment (mcranston18) [reply kitsune/users/jinja2/users/includes/login-form.html]: That URL is present in the target branch, but not in this feature branch. My plan was to hook this page up to the proper URLs, and create new ones where necessary, once this screen is merged.

Alternatively, if you'd rather, I can rebase this branch on top of the target and make the change.

Comment (johngian) [reply kitsune/users/jinja2/users/includes/login-form.html]: Let's do that

File: kitsune/users/jinja2/users/includes/login-ffx.html [hunk 3]
```diff
@@ -0,0 +1,20 @@
+{% from "users/includes/macros.html" import login_form as render_login with context %}
+
+<article id="login" class="main login-form">
+  <h1 class="mb-0">{{ _('Log in / Sign up with Firefox Accounts') }}</h1>
+  <p>
+    {{ _('We are upgrading our log-in system to replace Mozilla Support accounts with Firefox Accounts. Firefox Accounts offers one log-in for convenient and secure access to Mozilla services.') }} <a href="https://support.mozilla.org/en-US/kb/firefox-accounts-mozilla-support-faq">{{ _('See our FAQ') }}</a> {{ _('for more information') }}.
```
Comment (akatsoulas) [inline kitsune/users/jinja2/users/includes/login-ffx.html]: Let's wrap this line to approximately 100 to 120 chars.

File: kitsune/users/jinja2/users/includes/login-ffx.html [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+  <div>
+    <a href="{{ url('users.fxa_authentication_init') }}" class="button">{{ _('Continue with Firefox Accounts') }}</a>
+  </div>
+
+  <p>
+    <a id="legacy-login-toggle" href="#">{{ _('continue w/ SUMO account') }}</a>
+  </p>
+
+  <p class="help-text">
+    {{ _('Continuing with Firefox Accounts means you have agreed with the <a href="https://www.mozilla.org/en-US/about/legal/terms/services/">Term and Conditions</a> and <a href="https://www.mozilla.org/en-US/privacy/firefox/">Privacy Policy</a>' | safe )}}.
```
Comment (akatsoulas) [inline kitsune/users/jinja2/users/includes/login-ffx.html]: same here

File: kitsune/users/jinja2/users/includes/login-form.html [hunk 5]
```diff
@@ -1,6 +1,7 @@
 {% from "users/includes/macros.html" import login_form as render_login with context %}
 
-<article id="login" class="main">
-  <h1>{{ _('Sign In') }}</h1>
+<article id="login" class="main login-form">
+  <h1 class="mb-0">{{ _('Log In') }}</h1>
+  <p class="mt-0 mb-0">or <a href="{{ url('users.fxa_authentication_init') }}">{{ _('continue with Firefox Account') }}</a></p>
```
Comment (akatsoulas) [inline kitsune/users/jinja2/users/includes/login-form.html]: This link should display the FxA login screen and not start the auth flow.

**Comment by mcranston18** [comment]
@akatsoulas Just pushed up your fixes. For this PR, could you `Squash and Merge` in Github if approved? I can rebase but this PR includes merge commits from branches that _were_ squashed so rebasing would become more complicated

**Comment by akatsoulas** [comment]
@mcranston18 sounds good. FYI tests are failing

**Pull request merged**